tests/utils/paper_helpers.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+def compare_formatted_citations(expected: str, actual: str) -> bool:
+    """
+    Compares two formatted citation strings; ignoring the citation_count value.
+
+    :param expected: The expected formatted citation string.
+    :param actual: The actual formatted citation string.
+    :return: True if the citations match except for the citation count, False otherwise.
+    """
+    # https://regex101.com/r/lCN8ET/1
+    citation_pattern = r"(This article has )\d+( citations?)"
```

Comment:
This regex is closely coupled to `DocDetails`. Can you move this regex to a `ClassVar[str]` on `DocDetails`?

Reply:
done!